beta-4-setup.md

```diff
@@ -412,7 +412,7 @@ If you're feeling lazy, use your friend `sed`:
 
     sed -i -e 's/name: anypassword/name: apache_auth/' \
     -e 's/kind: AllowAllPasswordIdentityProvider/kind: HTPasswdPasswordIdentityProvider/' \
-    -e '/kind: HTPasswdPasswordIdentityProvider/i \      file: \/etc\/openshift-passwd' \
+    -e '/kind: HTPasswdPasswordIdentityProvider/i \      file: \/etc\/openshift/htpasswd' \
```

Comment:
Actually, are we certain there aren't going to be escaping issues here with `file: \/etc\/openshift/htpasswd`?